components/sections/Tabs/Tabs.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../../styles/fonts.scss";
+
+.tabs {
+  min-height: 300px;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  font-family: $poppins-semibold;
+  background-color: #31683d;
+}
+
+.tabsName {
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 30px;
+  justify-content: center;
+  align-content: center;
+  @media (max-width: 500px) {
+    flex-direction: column;
+  }
+}
+
+.tabTitle {
+  color: #ffffff;
+}
+
+.tabName {
+  font-size: 20px;
+  cursor: pointer;
+  margin-right: 30px;
+  color: #81a88f;
+  margin-left: 30px;
+  padding-bottom: 5px;
+}
+
+.tabsContent {
+  padding-left: 15vw;
```

Comment:
`padding: 0 15vw` 

i know im being a pain in the ass its just a recommendation to have simpler css but you can leave it like this if you want

Reply:
That's a perfectly valid comment, thanks!